### DISCUSSION TITLE
Documentation Code examples not displaying

### DISCUSSION THREAD

**Discussion by basscl** [webdriverio/webdriverio#11762] [Community]
E.g. on this page https://webdriver.io/docs/selectors a bunch of spots where I assume there is meant to be a code example it just displays a github blob link, e.g.:
```
To query an element with a specific tag name, use <tag> or <tag />.

https://github.com/webdriverio/example-recipes/blob/e8b147e88e7a38351b0918b4f7efbd9ae292201d/selectors/example.html#L5
```
Additionally if you follow the github link the blob you get doesn't always seem relevant to the issue being referenced.

It would be super helpful if the code examples worked!

**Comment by christian-bromann** [comment]
Thanks for reporting. This is a bug in our docs page and I will take a look asap, follow: https://github.com/webdriverio/webdriverio/issues/11763

**Reply by christian-bromann** [reply]
@basscl we have just redeployed the website with a fix. All examples should be displayed again.
